mini.go

```diff
@@ -137,19 +143,32 @@ func GET(writer http.ResponseWriter, reader *http.Request) {
 //the function which handle the post method
 //post the json data from broswer to the server and sql databases
 func Poster(w http.ResponseWriter, r *http.Request) {
+	//time when do request
+	RequestStart := time.Now()
 
 	//get the raw bytes of input data
 	bytes, errs := ioutil.ReadAll(r.Body)
 	if errs != nil {
-		fmt.Println(errs)
+		errString := fmt.Sprintf("buffer overflow %s", errs)
```

Comment:
You should not assume the error is a buffer overflow, just use the error message you get from `errs.Error()`
like this: `errString := errs.Error()`